phantom_token.c

```diff
@@ -671,14 +674,31 @@ static ngx_int_t introspection_response_handler(ngx_http_request_t *request, voi
     size_t jwt_len = request->headers_out.content_length_n;
     size_t bearer_jwt_len = BEARER_SIZE + jwt_len;
 
+    // Investigating upstream properties but currently I cannot reproduce the original error
```

Comment:
Currently I am monitoring these values, yet I am unable to reproduce the original server_error so we do not want to base code on upstream buffers until we are sure we need to, since it is a non-trivial change.